package.json

```diff
@@ -52,8 +52,8 @@
     "@types/jest-diff": "^24.3.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
-    "@typescript-eslint/eslint-plugin": "2.19.0",
-    "@typescript-eslint/parser": "2.19.0",
+    "@typescript-eslint/eslint-plugin": "2.34.0",
+    "@typescript-eslint/parser": "2.34.0",
```

Comment:
This was necessary to fix a parsing issue with TypeScript tuples (used in a test).